symfony/templates/notice/index.html.twig

```diff
@@ -19,9 +19,9 @@
             <tr>
                 <td><a href="{{ path('admin_notice', { id: notice.id }) }}">{{ notice.url }}</a></td>
                 <td>{{ notice.message|length > 100 ? notice.message|slice(0, 100) ~ '...' : notice.message  }}</td>
-                <td>{{ notice.full_page ? 'Yes' : 'No' }}</td>
-                <td>{{ notice.start_ts ? notice.start_ts|date('n/j/Y g:ia', app.userTimezone) }}</td>
-                <td>{{ notice.end_ts ? notice.end_ts|date('n/j/Y g:ia', app.userTimezone) }}</td>
+                <td>{{ notice.fullPage ? 'Yes' : 'No' }}</td>
+                <td>{{ notice.startTs ? notice.startTs|date('n/j/Y g:ia', app.userTimezone) }}</td>
+                <td>{{ notice.endTs ? notice.endTs|date('n/j/Y g:ia', app.userTimezone) }}</td>
```

Comment:
Don't we need to switch these to `app.user.info.timezone`?

Reply:
Switched them to `app.user.timezone` with the fix in 39524cb.